data/stylesheets/asciidoctor-default.css

```diff
@@ -295,7 +295,7 @@ sup.footnote a:active,sup.footnoteref a:active{text-decoration:underline}
 #footnotes .footnote:last-of-type{margin-bottom:0}
 #content #footnotes{margin-top:-.625em;margin-bottom:0;padding:.75em 0}
 div.page-break{display:none}
-div.unbreakable{page-break-inside:avoid}
+div.unbreakable{-moz-column-break-inside:avoid;break-inside:avoid}
```

Comment:
According to MDN, Firefox supports `break-inside:avoid`. Do we really need `-moz-column-break-inside`?

https://developer.mozilla.org/en-US/docs/Web/CSS/break-inside

Reply:
That's being added by autoprefixer. It seemed odd to me too, but I couldn't see how to make it stop without changing our setting from "default". See https://github.com/asciidoctor/asciidoctor/blob/main/src/stylesheets/package.json#L16-L18